google-cloud-kms/Gemfile

```diff
@@ -8,5 +8,5 @@ if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.1")
   # We think this is because of a bug in Bundler 1.6.
   # Specify a viable version to allow the build to succeed.
   gem "jwt", "~> 1.5"
-  gem "kramdown", "< 1.17.0" # Error in yard with 1.17.0
+  gem "kramdown", "< 1.17.1" # Error in yard with 1.17.0
```

Comment:
I don't get this. If YARD has errors in 1.17.0, why would we bump to `< 1.17.1` and allow 1.17.0? Why is Renovate asking for this change?

Reply:
My guess is Renovate doesn't understand the semantics of why we'd use only `< x.y.z`.

Reply:
Once we drop Ruby 2.3 we can probably remove all these lines.